Build failing via SPM: Syntax error (unexpected ',' separator) in Package.swift on last version 12.13.0

### Description

**Describe the bug**
There is a syntax error in the `Package.swift` file introduced in the latest release (12.13.0). A trailing comma after the `swiftSettings` argument in the `FirebaseAILogic` target causes the Swift compiler to fail when resolving package dependencies.

**To Reproduce**
Steps to reproduce the behavior:
1. Update `firebase-ios-sdk` via Swift Package Manager to version `12.13.0`.
2. Trigger a build (in my case, this occurred on our Bitrise CI pipeline).
3. The dependency resolution fails immediately with the following error: `error: unexpected ',' separator`.

**Expected behavior**
The `Package.swift` should be parsed correctly and dependencies should resolve without syntax errors.

**Additional context**
I validated that this is not an issue with my local project configuration. The Bitrise error logs point exactly to line 198 of the `Package.swift` file. 

Looking at the manifest, there is an invalid trailing comma after the `swiftSettings` array in the `FirebaseAILogic` target definition:

In Swift, trailing commas after the last argument in a function call are not allowed.

As a temporary workaround, I have locked my project's dependency to exact version 12.12.0, which compiles successfully. Could you please release a patch to fix this syntax issue? Thank you!

<img width="1082" height="597" alt="Image" src="https://github.com/user-attachments/assets/cd282ce1-f057-46e8-939e-a1ea343ae17f" />

<img width="577" height="191" alt="Image" src="https://github.com/user-attachments/assets/b85a745d-20d0-4bfd-bf04-6c2971879164" />

### Reproducing the issue

**To Reproduce**
Steps to reproduce the behavior:
1. Update `firebase-ios-sdk` via Swift Package Manager to version `12.13.0`.
2. Trigger a build (in my case, this occurred on our Bitrise CI pipeline).
3. The dependency resolution fails immediately with the following error: `error: unexpected ',' separator`.

### Firebase SDK Version

12.13.0

### Xcode Version

26.4.1

### Installation Method

Swift Package Manager

### Firebase Product(s)

Infrastructure

### Targeted Platforms

iOS

### Relevant Log Output

```shell
xcodebuild: error: Could not resolve package dependencies:
  fatalError
  Unexpected ',' separator
  <unknown>:0: warning: legacy driver is now deprecated; consider avoiding specifying '-disallow-use-new-driver'
/Users/vagrant/Library/Developer/Xcode/DerivedData/app-skyline-ios-dpvgoqweamwcgubxdkzrdtoewhqi/SourcePackages/checkouts/firebase-ios-sdk/Package.swift:199:5: error: unexpected ',' separator
 197 |         isFoundationModelsSupportedPlatformSwiftSetting(),[0m
 198 |       ],[0m
 199 |     ),[0m
     |     `- error: unexpected ',' separator
 200 |     .testTarget([0m
 201 |       name: "FirebaseAILogicUnit",[0m

Exit status: 74
```

### If using Swift Package Manager, the project's Package.resolved

<!--- Look below for instructions on how to share your Package.resolved. --->

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>


### If using CocoaPods, the project's Podfile.lock

<!--- Look below for instructions on how to share your Podfile.lock. --->

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

--- a/FirebaseCore/CHANGELOG.md
+++ b/FirebaseCore/CHANGELOG.md
@@ -1,5 +1,5 @@
 # Unreleased 
-- [fixed] Remove extra command in Package.swift that caused SPM resolution
+- [fixed] Remove extra comma in Package.swift that caused SPM resolution
   failures for some configurations.
 
 # Firebase 12.12.0

--- a/FirebaseCore/CHANGELOG.md
+++ b/FirebaseCore/CHANGELOG.md
@@ -1,3 +1,7 @@
+# Unreleased 
+- [fixed] Remove extra command in Package.swift that caused SPM resolution
+  failures for some configurations.
+
 # Firebase 12.12.0
 - [changed] Firebase now requires at least Xcode 26.2 and the Swift
   6.2.3+ toolchain. This update allows Firebase to leverage new Swift

--- a/FirebaseCore/CHANGELOG.md
+++ b/FirebaseCore/CHANGELOG.md
@@ -1,4 +1,4 @@
-# Unreleased 
+# Unreleased
 - [fixed] Remove extra comma in Package.swift that caused SPM resolution
   failures for some configurations.
 

--- a/Package.swift
+++ b/Package.swift
@@ -195,7 +195,7 @@ let package = Package(
       path: "FirebaseAI/Sources",
       swiftSettings: [
         isFoundationModelsSupportedPlatformSwiftSetting(),
-      ],
+      ]
     ),
     .testTarget(
       name: "FirebaseAILogicUnit",

